Woof/WoofTests/Model/LocalStorage/LocalStorageDeleteTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+
+final class LocalStorageDeleteTests: XCTestCase {
+    var localStorage: LocalStorage!
```

Comment:
Why does the current variable have the internal visibility?

Reply:
fixed